pkg/config/utils/tags.go

```diff
@@ -5,7 +5,28 @@
 
 package utils
 
-import "github.com/DataDog/datadog-agent/pkg/config"
+import (
+	"sort"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// removeDuplicatesAndSort sorts and removes duplicates from a slice without doing it in place.
+// The final slice should have len(res) == len(res). It is already defined in pkg/util but can not be imported
```

Comment:
Instead of `len(res) == len(res)`, did you mean `cap(res) == len(res)` ?

Because, the first assertion is obvious and the second one is false: https://go.dev/play/p/7zAnlIyFtFq

```go
func main() {
	s := removeDuplicatesAndSort([]string{"foo", "bar", "baz"})
	fmt.Printf("s=%#v, len(s)=%d, cap(s)=%d", s, len(s), cap(s))
}
```

```
s=[]string{"bar", "baz", "foo"}, len(s)=3, cap(s)=4
```

Reply:
Nice catch. It was a typo here